lib/oembed/providers/builtin_providers.rb

```diff
@@ -6,6 +6,7 @@
 require 'oembed/providers/facebook_video'
 require 'oembed/providers/instagram'
 require 'oembed/providers/tiktok'
+require 'oembed/providers/matterport'
```

Comment:
**Nit:** I have a slight preference for keeping these lines in alphabetical order (unless there's a specific reason not to)

```suggestion
require 'oembed/providers/matterport'
require 'oembed/providers/tiktok'
```

Reply:
I agree with that - fixed 👍